apps/backoffice-v2/src/pages/Entity/components/Case/components/CaseOptions/hooks/useCaseOptionsLogic/useCaseOptionsLogic.tsx

```diff
@@ -22,5 +22,6 @@ export const useCaseOptionsLogic = () => {
     generateAndOpenPDFInNewTab,
     isCopyingCollectionFlowLink,
     copyCollectionFlowLink,
+    isDemo: customer?.config?.isDemo,
```

Comment:
We have two flags for Demo flags, make sure the logic is activated if either flag is present